ambari-server/src/main/java/org/apache/ambari/server/state/cluster/ClusterImpl.java

```diff
@@ -2418,7 +2435,9 @@ public void applyLatestConfigurations(StackId stackId) {
 
       // !!! without providing the flush here, when this transaction completes it
       // looks like the database has all unselected configs for some types
-      clusterEntity = clusterDAO.merge(clusterEntity, true);
+      for (ClusterConfigEntity clusterConfig : configEntities) {
```

Comment:
we using this structure multiple times, why not create something like `clusterDAO.merge(List<ClusterConfigEntity> entities, boolean flush)`?